docs/themes.md

```diff
@@ -53,6 +53,16 @@ theme:
   primary-color: 97 13 80
 ```
 
+### Kanagawa Dark
+![screenshot](images/themes/kanagawa-dark.png)
+```yaml
+theme:
+  background-color: 240 13 14
+  priamry-color: 51 33 68
+  negative-color: 359 52 31
```

Comment:
Could we use a lighter negative color? The contrast seems pretty low with the current one. 

https://coolors.co/contrast-checker/782627-1f1f28

Reply:
Does [\#FF5D62](https://coolors.co/contrast-checker/ff5d62-1f1f28) fit better?

Reply:
Yeah much better 👍🏻

Reply:
Alright, [I've added the colors in the config file](https://github.com/glanceapp/glance/pull/80/commits/d27993d2c954bd9925cd8493eb7498e7897b945d). I'll add an updated screenshot of the theme to go along with it as soon as I can.